modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/help.text/HelpTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.field.type.internal.help.text;
+
+import com.liferay.dynamic.data.mapping.form.field.type.DDMFormFieldTemplateContextContributor;
+import com.liferay.dynamic.data.mapping.form.field.type.constants.DDMFormFieldTypeConstants;
+import com.liferay.dynamic.data.mapping.model.DDMFormField;
+import com.liferay.dynamic.data.mapping.model.LocalizedValue;
+import com.liferay.dynamic.data.mapping.render.DDMFormFieldRenderingContext;
+import com.liferay.petra.string.StringPool;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Bruno Basto
+ */
+@Component(
+	immediate = true,
+	property = "ddm.form.field.type.name=" + DDMFormFieldTypeConstants.HELP_TEXT,
+	service = {
+		DDMFormFieldTemplateContextContributor.class,
+		HelpTextDDMFormFieldTemplateContextContributor.class
+	}
+)
+public class HelpTextDDMFormFieldTemplateContextContributor
+	implements DDMFormFieldTemplateContextContributor {
+
+	@Override
+	public Map<String, Object> getParameters(
+		DDMFormField ddmFormField,
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
+
+		return Collections.singletonMap(
+			"text", getText(ddmFormField, ddmFormFieldRenderingContext));
+	}
+
+	protected String getText(
+		DDMFormField ddmFormField,
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
+
+		LocalizedValue text = (LocalizedValue)ddmFormField.getProperty("text");
+
+		if (text == null) {
+			return StringPool.BLANK;
+		}
+
+		return text.getString(ddmFormFieldRenderingContext.getLocale());
+	}
```

Comment:
@mateusgomes01 Do you really need this class? Do you need this “text” property in the front-end code?? If not, please remove this class.

Reply:
Sure thing! Removing them.